### PR TITLE
Enhacements

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

- [x] Update workflows github/actions.
- [x] Add config `AuthenticationMethodInterface::class`.
- [x] Update README.md.

### CODE REVIEW

**Review by xepozz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This [Yii Framework] extension allows easy creation of web applications.
 [![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/yii-web/badges/coverage.png?s=31d80f1036099e9d6a3e4d7738f6b000b3c3d10e)](https://scrutinizer-ci.com/g/yiisoft/yii-web/)
 [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Fyii-web%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/yii-web/master)
 [![static analysis](https://github.com/yiisoft/yii-web/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/yii-web/actions?query=workflow%3A%22static+analysis%22)
-
+[![type-coverage](https://shepherd.dev/github/yiisoft/yii-web/coverage.svg)](https://shepherd.dev/github/yiisoft/yii-web)
```
Comment (samdark) [inline README.md:22]: What's that?

**Comment by terabytesoftw** [comment]
@samdark is there any package that uses bower that is defined as aliases in yii-web?

**Comment by samdark** [comment]
@terabytesoftw I don't think so.

**Comment by samdark** [comment]
👍

**Pull request merged**